proxy/src/heartbeat.rs

```diff
@@ -2,7 +2,6 @@ use std::{
     net::SocketAddr,
     sync::{
         atomic::{AtomicBool, Ordering},
-        Arc,
```

Comment:
this is breaking

Reply:
Fixed in https://github.com/jito-labs/shredstream-proxy/pull/40/commits/9a21c9157006c7731a6a6dac16d2da8770e29c10